### PR TITLE
Add suse support to ansible (collector only)

### PR DESCRIPTION
**Pull request by jcheng-splunk**
Also updated molecule tests for custom opensuse images and to skip fluentd tests when running on suse.

### CODE REVIEW

**Review by emaderer — APPROVED** [review]

File: deployments/ansible/molecule/default/Dockerfile.j2 [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+{% else %}
+FROM opensuse/leap:15
+RUN zypper -n install -l ansible dbus-1 python3-rpm sudo systemd-sysvinit
+{% endif %}
+
+ENV container docker
+
+RUN (cd /usr/lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i = \
+    "systemd-tmpfiles-setup.service" ] || rm -f $i; done); \
+    rm -f /usr/lib/systemd/system/multi-user.target.wants/*;\
```
Comment (rmfitzpatrick) [inline deployments/ansible/molecule/default/Dockerfile.j2:16]: Is this `/usr/lib/systemd` on all platforms or just suse?

Comment (rmfitzpatrick) [reply deployments/ansible/molecule/default/Dockerfile.j2:16]: oh, missed the parent`if`...

Comment (jcheng-splunk) [reply deployments/ansible/molecule/default/Dockerfile.j2:16]: Yeah, the idea was to use custom images for opensuse, and just use the pre-built `geerlingguy` ansible images for all other distros.

**Review by rmfitzpatrick — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]
LGTM

**Pull request merged**